src/ui/pages/edit/PagesEditFormContainer.js

```diff
@@ -42,7 +42,7 @@ export const mapDispatchToProps = dispatch => ({
         }
         default: history.push(ROUTE_PAGE_TREE);
       }
-    }),
+    }).catch(() => {}),
```

Comment:
no need to notify user here?

Reply:
it's already in `sendPutPage` method (line 332~) in `state/pages/actions.js`. I only added catch on that line there to avoid unhandled reject errors.